guides/common/modules/proc_comparing-content-view-versions.adoc

```diff
@@ -4,7 +4,9 @@
 Use this procedure to compare Content View Version functionality for {Project}.
 
 .Procedure
-* In the {ProjectWebUI}, navigate to *Content* > *Content Views*, then select two Content View versions and click *Compare*.
+. In the {ProjectWebUI}, navigate to *Content* > *Content Views*.
```

Comment:
Also, Content View needs to be selected first to compare its versions. Perhaps this is what the deleted step was intended for, in which case I apologize as I got confused by the previous formulation of the procedure. 
```suggestion
. In the {ProjectWebUI}, navigate to *Content* > *Content Views*.
. Select a Content View whose versions you want to compare.
```